.github/actions/make/web/ts/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Fetch or make the web bindings files and wrappers
+inputs:
+  gh-token:
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+      - name: download wasm binary
+        uses: ./.github/actions/make/web/wasm
+        with:
+          gh-token: ${{ inputs.gh-token }}
+
+      # ensure bun deps are installed freshly before any command using bun is invoked
+      - name: install bun deps
+        run: |
+          make bun-deps
+        shell: bash -leo pipefail {0}
```

Comment:
It would be nice to avoid specifying this in many places.

Can we make use of workflow defaults?
https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#defaultsrun
https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#defaultsrunshell